src/EventHub/EventHub.Autorest/examples/Get-AzEventHubPrivateEndpointConnection.md

```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
+```powershell
+{{ Add code here }}
```

Comment:
Please fill these with relevant examples, you should find some examples https://learn.microsoft.com/en-us/powershell/module/az.eventhub/get-azeventhubprivateendpointconnection?view=azps-11.3.0